anchor/common/ssv_types/src/operator.rs

```diff
@@ -0,0 +1,14 @@
+/// Unique identifier for an Operator
+pub type OperatorID = u64;
```

Comment:
For things that we will be using a lot, we should use the new type idiom: https://doc.rust-lang.org/rust-by-example/generics/new_types.html

This enforces the type at compile time, so we can't get confused by accidentally putting another type that is a u64 in place of this. 

For the operator-id, I have already done this. See [here](https://github.com/sigp/anchor/blob/stable/anchor/qbft/src/types.rs#L53) - I used usize, (I didn't check the specs :p, if the specs say u64, then we should change it to that).